app/controllers/state_file/questions/retirement_income_subtraction_controller.rb

```diff
@@ -5,6 +5,10 @@ class RetirementIncomeSubtractionController < QuestionsController
 
```

Comment:
Do we still need to include `RepeatedQuestionConcern`

Reply:
We don't _need_ it, no - we could move that code into this file, since it's the only place we _currently_ use `RepeatedMultiPageStep`. My reason not for doing so is that while it's a tiny amount of code, it'll hopefully be a helpful starting point (that'll help us avoid some code duplication) for whatever controller we create next that isn't 1099R-related but is used within a `RepeatedMultiPageStep`. Do you have an idea for an alternative approach, or a reason you'd like to see it removed?

Reply:
That makes sense. I think it is fine here.